src/intervals/intervals.jl

```diff
@@ -36,6 +37,8 @@ struct Interval{T<:Real} <: AbstractInterval{T}
     end
 end
 
+@inline _normalisezero(a::Real) = ifelse(iszero(a) && signbit(a), copysign(a, 1), a)
+@inline _normalisezero(a::Integer) = float(a)
```

Comment:
is this needed? I think integer is already converted to float by the constructor at some point?

Reply:
The first one yes; the second one I am not yet sure... There is a corner case I'm trying to figure out...

Reply:
I'm checking it right now. The subtlety with that line is that `Integer`s have no signbit, which was the reason I original included it.

Reply:
the constructor for integers is
```julia
Interval(a::T, b::T) where T<:Integer = Interval(float(a), float(b))
```
so I think the integer is converted to float before getting to that line.

Reply:
You are right... I'll get rid of it.